lib/build/bundle.js

```diff
@@ -194,6 +195,30 @@ exports.Bundle = class {
     });
   }
 
+  copyFiles() {
+    if (!this.config.dependencies) {
+      return Promise.resolve();
+    }
+
+    const output = this.bundler.project.platform.output;
+    const root = this.bundler.project.paths.root;
+    let filesToCopyList = this.config.dependencies
+      .filter(x => x instanceof Object && x[copyProperty]);
```

Comment:
@AStoker this is another thing that is bothering me is this. To be honest, I don't think this is common composition. Do you know if there is a problem using instanceof Object ? typeof x == 'object' seems to be more common.

Any ideas?